docs/source/_templates/page.html

```diff
@@ -62,7 +68,8 @@
       <div class="theme-toggle-container theme-toggle-header">
         <button class="theme-toggle">
           <div class="visually-hidden">Toggle Light / Dark / Auto color theme</div>
-          <svg class="theme-icon-when-auto"><use href="#svg-sun-half"></use></svg>
+          <svg class="theme-icon-when-auto-light"><use href="#svg-sun-half"></use></svg>
+          <svg class="theme-icon-when-auto-dark"><use href="#svg-sun-half"></use></svg>
```

Comment:
Aligning with furo ([ref](https://github.com/pradyunsg/furo/blob/1bbf4ee4f65c488b9a0f577b8205255338a64c4d/src/furo/theme/furo/page.html#L45-L46))

```suggestion
          <svg class="theme-icon-when-auto-light"><use href="#svg-sun-with-moon"></use></svg>
          <svg class="theme-icon-when-auto-dark"><use href="#svg-moon-with-sun"></use></svg>
```

We'll also need to sync with furo's [icons.html](https://github.com/pradyunsg/furo/blob/1bbf4ee4f65c488b9a0f577b8205255338a64c4d/src/furo/theme/furo/partials/icons.html) file.

Reply:
Added!